platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -148,6 +144,8 @@ typedef struct crypto_local_t {
 
 static __thread crypto_local_t local;
 
+static odp_crypto_packet_result_t *get_op_result_from_packet(odp_packet_t pkt);
```

Comment:
Definition of `get_op_result_from_packet()` could be just moved before `set_crypto_op_result()` as it is now used in that function for the first time.

Reply:
Will move it.